app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/walks/WalkFragment.java

```diff
@@ -56,9 +64,13 @@
     private WalkPets selectedWalkPets;
     private int[] colors;
     private int nextColor;
+    private static FloatingActionButton flSharePetWalkRouteButton;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Static variable definition in wrong order.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5631707)